fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/TrackedDataHandlerRegistryMixin.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.object.builder;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.entity.data.TrackedDataHandlerRegistry;
+
+import net.fabricmc.fabric.impl.object.builder.FabricTrackedDataRegistryImpl;
+
+@Mixin(TrackedDataHandlerRegistry.class)
+public class TrackedDataHandlerRegistryMixin {
```

Comment:
Would it be worth while injecting into `register` and logging a warning (or even crashing in dev?) if a handler is registered the vanilla way? 

Logging the warning with a stacktrace would be more helpful when trying to track down mods that arent using the API.